Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -57,7 +58,7 @@ public static string GetMask(DependencyObject obj)
         /// </summary>
         /// <param name="obj">TextBox Control</param>
         /// <param name="value">Mask Value</param>
-        public static void SetMask(DependencyObject obj, string value)
+        public static void SetMask(TextBox obj, string value)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind